httpserver/src/main/kotlin/tbdex/sdk/httpserver/TbdexHttpServer.kt

```diff
@@ -69,8 +68,8 @@ class TbdexHttpServerConfig(
  * @property config The configuration for the server, including port and optional APIs.
  */
 class TbdexHttpServer(val config: TbdexHttpServerConfig) {
-  val getCallbacks: MutableMap<String, GetCallback> = mutableMapOf()
-  val submitCallbacks: MutableMap<String, SubmitCallback> = mutableMapOf()
+  // todo i don't want to use Any :/
```

Comment:
Is this a placeholder that's going to get solved in this PR or a future one?

Reply:
this PR - just solved it with help from @diehuxx  :)